client/client-multi/client-cpp/test/impl/ClientStatusTest.cpp

```diff
@@ -31,18 +40,27 @@ const char * const filename = RESOURCE_DIR"/kaa_status.file";
 const char * const filename = "kaa_status.file";
 #endif
 
+
 void cleanfile() {
     std::remove(filename);
 }
 
 namespace kaa {
 
+static MockExecutorContext context;
+static KaaClientProperties properties;
+static DefaultLogger tmp_logger(properties.getClientId());
+
 BOOST_AUTO_TEST_SUITE(ClientStatusSuite);
 
 BOOST_AUTO_TEST_CASE(checkDefaults)
 {
     cleanfile();
 
+//    auto stateMock = std::make_shared<MockKaaClientStateStorage>();
+//    properties.setStateFileName(filename);
+//    KaaClientContext clientContext(properties, tmp_logger, *stateMock, context);
+
```

Comment:
Do not leave commented code for no reason. You always can use the SCM to restore any piece of code that you want.